platform/server-config/src/main/java/org/motechproject/config/service/impl/ConfigurationServiceImpl.java

```diff
@@ -308,7 +308,7 @@ public void addOrUpdateProperties(String bundle, String version, String filename
             //Persist only non-default properties in database
             toPersist = new Properties();
             for (Map.Entry<Object, Object> entry : newProperties.entrySet()) {
-                if (!defaultProperties.containsKey(entry.getKey()) ||
+                if (defaultProperties == null || !defaultProperties.containsKey(entry.getKey()) ||
```

Comment:
why would this method get called for json files?  Isn't this only for property files?